sw/simulator/nps/nps_main.c

```diff
@@ -356,8 +356,7 @@ static bool nps_main_parse_options(int argc, char **argv)
           case 2:
             nps_main.fg_time_offset = atoi(optarg); break;
           case 3:
-            if (optarg == NULL) {nps_main.js_dev = strdup("0");}
-            else {nps_main.js_dev = strdup(optarg);}
+            nps_main.js_dev = strdup(optarg);
```

Comment:
Hmm, it doesn't or at least not the same way - marking as intentional, whatever the original reason here was...